### ISSUE TITLE
Firefox Account avatar element does not have [alt] attribute

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#835]
![Firefox_Monitor___Scan_Results](https://user-images.githubusercontent.com/557895/54240284-e0684580-44da-11e9-85d4-5df145122965.png)

```html
<div class="avatar-wrapper desktop hide-mobile">
  <img class="avatar" src="https://firefoxusercontent.com/a85549d62dfa9e46893f311e04c2a0aa">
</div>
```

**Comment by pdehaan** [comment]
Not sure what the `alt=""` should be. The FxA display name? The email address?

**Comment by lesleyjanenorton** [comment]
I think the email address is possible safer since some users may not have a display name.
